### PR TITLE
Balances page

### PR DESCRIPTION
**Pull request by FNKMYR**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -46,4 +59,21 @@ export default function App() {
...
+        .split('')
+        .reverse()
+        .reduce(function (acc, num, i, orig) {
+          return num === '-' ? acc : num + (i && !(i % 3) ? ',' : '') + acc;
+        }, '') +
+      '.' +
+      p[1] +
+      ' €'
+    );
+  }
```
Comment (mwarnck) [inline src/App.js:78]: I don´t understand a single line here but it looks very useful and fancy :)

File: src/components/Balance.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        </span>
+        <span>
+          <StyledSpanMedium>
+            Received: {expenseFormatter(moneyReceived)}
+          </StyledSpanMedium>
+        </span>
+      </Middle>
+      <RightSide>
+        {balance === 0 ? (
+          <StyledSpan18>{expenseFormatter(balance)}</StyledSpan18>
```
Comment (mwarnck) [inline src/components/Balance.js:34]: Why StyledSpan18?

File: src/components/BalanceList.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  return (
+    <Wrapper>
+      <TotalExpenses>
+        <p>Total expenses:</p>
+        <p>{expenseFormatter(totalExpenses)}</p>
+      </TotalExpenses>
+      <Scrollarea>
+        {members.map((member, index) => (
+          <Balance
+            key={index}
```
Comment (mwarnck) [inline src/components/BalanceList.js:27]: Maybe there is another value for the keys available which is unique and better than the index?

File: src/components/BalanceList.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        .reverse()
+        .reduce(function (acc, num, i, orig) {
+          return num === '-' ? acc : num + (i && !(i % 3) ? ',' : '') + acc;
+        }, '') +
+      '.' +
+      p[1] +
+      ' €'
+    );
+  }
+}
```
Comment (mwarnck) [inline src/components/BalanceList.js:53]: This is the same function like in the App.js when I am right. Isn´t it possible to handle it as prop to the BalanceList to reduce code?

File: src/components/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    background: ${props => props.theme.color.secondaryDark};
+  }
+`;
+
+const Expenses = styled(NavLink)`
+  &.active {
+    color: ${props => props.theme.color.complementaryLight};
+    background: ${props => props.theme.color.secondaryDark};
+  }
+`;
```
Comment (mwarnck) [inline src/components/Navigation.js:66]: This code is the same. You do not have to rename the NavLinks and could style them inside your wrapper or give them both the same name for the styled components stylings to reduce code.

File: src/components/Navigation.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+export default function Navigation({ expensesActive }) {
+  return (
+    <Wrapper>
+      <Expenses to="/">Expenses</Expenses>
+      <Balances to="/balances">Balances</Balances>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.header`
```
Comment (mwarnck) [inline src/components/Navigation.js:13]: I am not sure if header is the right element here, while you place the navigation on the BalancesPage at the bottom of the page which also includes a header element.

**Review by mwarnck — COMMENTED** [review]
Overall great work! I left a few comments you should have a look at :)

File: src/components/BalanceList.js (hunk 4 above)
Comment (FNKMYR) [reply src/components/BalanceList.js:53]: Pulled it up and forgot to delete it here. The App.js one was the only one in use already.

File: src/components/Navigation.js (hunk 6 above)
Comment (FNKMYR) [reply src/components/Navigation.js:13]: Good point. Again, duplicated a component and missed something.

**Pull request merged**